_posts/sumtyper.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: post
+title: "Sumtyper"
+---
+
+# Modellering med varianter/sumtyper/sealed classes
```

Comment:
Klarer du å finne en overskrift som tydeligere sier hva man kan oppnå? Kanskje et hakk mer tabloid og litt mindre deskriptiv?

Reply:
Ett forslag kan være `Reduser kompleksitet med bedre datamodeller (i Java)`? Tipper i alle fall at "verdiforslaget" trekker mer interesse for målgruppen enn stikkord som "sumtyper" og "sealed classes".